packages/eslint-plugin-codemod/lib/index.ts

```diff
@@ -2,6 +2,7 @@ import renameProp from './rules/rename-prop'
 import changeComposition from './rules/change-composition'
 import sortImports from './rules/sort-imports'
 import noCodemodComment from './rules/no-codemod-comment'
+import rule from '@thing/button/rule'
```

Comment:
nothing stopping a rule being created cross module boundaries and consumed in a single plugin